assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -212,9 +212,14 @@ che.docker.ip.external=NULL
 #   - 'docker-local': internal address is address of container within docker network, and exposed ports
 #                     are used.
 #   - 'custom': The evaluation strategy may be customized through a template property.
+#   - 'single-port':  internal address is set as in docker-local strategy, external address is composed
```

Comment:
I think the single port could reuse the custom strategy as the expression "serverName-workspaceID-cheExternalAddress" can be addressed by using the  template <serverName>-<workspaceId>-< externalAddresss>